nodes/pods/nodes-pods-user-namespaces.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: nodes-pods-user-namespaces
+[id="nodes-pods-user-namespaces"]
+= Running pods in Linux user namespaces
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+Linux user namespaces allow administrators to isolate the container user and group identifiers (UIDs and GIDs) so that a container can have a different set of permissions in the user namespace than on the host system where it is running. This allows containers to run processes with full privileges inside the user namespace, but the processes can be unprivileged for operations on the host machine.
+
+By default, a container runs in the host system's root user namespace. Running a container in the host user namespace can be useful when the container needs a feature that is available only in that user namespace. However, it introduces security concerns, such as the possibility of container breakouts, in which a process inside a container breaks out onto the host where the process can access or modify files on the host or in other containers. 
+ 
+Running containers in individual user namespaces can mitigate contiainer breakouts and several other vulnerabilities that a compromised container can pose to other pods and the node itself. 
+
+You can configure Linux user namespace use by setting the `hostUsers` parameter to `false` in the pod spec, as shown in the following procedure.
+
+:FeatureName: Support for Linux user namespaces
```

Comment:
Does :FeatureName: do anything here? It seems to be doing nothing?

Reply:
The :FeatureName: feeds the string that follows into the Technology Preview snippet. See the [Important note here](https://81592--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-user-namespaces.html).